src/plugins/gui/scenes/EnterAmountScene.js

```diff
@@ -157,7 +157,7 @@ export const FiatPluginEnterAmountScene = memo((props: Props): React.Node => {
 
                   <View style={styles.poweredByContainerColumn}>
                     <View style={styles.poweredByContainerRow}>
-                      <Text style={styles.poweredByText}>{s.strings.plugin_powered_by}</Text>
+                      <Text style={styles.poweredByText}>{s.strings.plugin_powered_by_space}</Text>
```

Comment:
It appears there is already a space char below? Also why isn't `%s` used here and only one `Text` component?

Reply:
Good catch